projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.css

```
@@ -456,33 +456,75 @@
 	}
 }
 
-/* Fixes the order of screen options,
- * showing the dashboard switcher button always first.
-*/
-#adv-settings {
-	display: flex;
-	flex-direction: column;
+/* Dashboard Switcher */
+#view-link-wrap {
+	float: left;
+	margin: 0 0 0 6px;
+}
+
+.screen-options-tab__wrapper {
+	position:relative
```

Nitpick: It'd be great to follow the [WordPress CSS Coding Standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/css/).
```suggestion
	position: relative;
```